messaging/counter/package.json

```diff
@@ -26,7 +26,7 @@
     "@types/node": ">=12.0.0",
     "@typescript-eslint/eslint-plugin": "^5.59.9",
     "@typescript-eslint/parser": "^5.59.9",
-    "@zetachain/toolkit": "^4.0.0",
+    "@zetachain/toolkit": "5.0.0",
```

Comment:
why fixed and not ^?

Reply:
Some odd `yarn` behaviour, fixed: https://github.com/zeta-chain/example-contracts/pull/99/commits/f4bd7cc348d66bd3cc4535d8e700cecc27ceb9a9

It's indeed better to have `^5.0.0`, because we might want to ship fixes that should be automatically installed.